.github/workflows/go.yml

```diff
@@ -0,0 +1,59 @@
+name: Go
+
+on:
+  push:
+    branches: [ "xinyu/httpserver" ]
+  pull_request:
+    branches: [ "xinyu/httpserver" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.20'
+    #test below failed with the following errors:
+      #Error: api/plugin/queue/queuehost.go:39:11: undefined: Queries
+      # Error: api/plugin/queue/queuehost.go:145:14: undefined: New
+      # Error: api/plugin/queue/queuehost.go:178:11: undefined: CreateIdToKeyMappingParams
+      # Error: api/plugin/queue/queuehost.go:201:3: undefined: getKeyFromQueueIdParams
+      # Error: api/plugin/queue/queuehost.go:224:7: undefined: getKeyFromQueueIdParams
+      # Error: api/plugin/queue/queuehost.go:296:13: undefined: CreateMessageParams
+      # Error: api/plugin/queue/queuehost.go:370:8: undefined: MarkDoneParams
+      # Error: api/plugin/queue/queuehost.go:428:7: undefined: RetrieveMessageParams
+      # Error: api/plugin/queue/queuehost.go:500:9: undefined: UpdateMessageRetrievedParams
+      # Error: Process completed with exit code 1.
+    # - name: Plugin Queue Test
+    #   run: go test -v github.com/iansmith/parigot/api/plugin/queue
+      # test below failed with the following errors:
+      #   Error: api/plugin/file/filehost.go:251:21: undefined: file.FileErr_OpenError
+      #   Error: api/plugin/file/filehost.go:306:22: undefined: file.FileErr_CreateError
+      #   Error: api/plugin/file/filehost.go:397:21: undefined: file.FileErr_ReadError
+      #   Error: api/plugin/file/filehost.go:406:21: undefined: file.FileErr_EOFError
+      #   Error: api/plugin/file/filehost.go:446:22: undefined: file.FileErr_DeleteError
+      #   Error: api/plugin/file/filehost.go:499:21: undefined: file.FileErr_WriteError
+      #   Error: api/plugin/file/file_test.go:140:20: undefined: file.FileErr_EOFError
+      #   Error: Process completed with exit code 1.
+    - name: Plugin File Test
```

Comment:
Xinyu's recent change fixes this.

Can you get rid of that mass of comments showing what the error is?   It was kinda confusing to me at first.